src/content/docs/alerts-applied-intelligence/new-relic-alerts/alert-conditions/quick-start-nrql-condition.mdx

```diff
@@ -0,0 +1,51 @@
+---
+title: Write your first NRQL alert condition
+tags:
+  - Alerts and Applied Intelligence
+  - New Relic Alerts
+  - Alert conditions
+  - NRQL
+metaDescription: How to start writing NRQL alert conditions with a step by step user case.
+---
+
+Use [NRQL queries](/docs/query-your-data/nrql-new-relic-query-language/get-started/introduction-nrql-new-relics-query-language/) to create alert [conditions](/docs/alerts-applied-intelligence/new-relic-alerts/alert-conditions/create-alert-conditions/) with ease.
+
+Your condition only needs two attributes: a [query](/docs/query-your-data/explore-query-data/query-builder/introduction-query-builder/#specify-data) and a [threshold](/docs/alerts-applied-intelligence/new-relic-alerts/alert-conditions/set-thresholds-alert-condition/).
+
+Follow our step-by-step example to write your first alert condition using the NRQL query builder.
+
+<Callout variant="important">
+This is a quick guide for creating NRQL alert conditions. For more information on NRQL alert examples, tips, and troubleshooting, see [Create NRQL alert conditions](/docs/alerts-applied-intelligence/new-relic-alerts/alert-conditions/create-nrql-alert-conditions/). 
```

Comment:
On second thought (see my Jira comment), I don't think we need this callout. Or if we do keep it, I'd mention that you can only create a working condition (or even a valid NRQL query) if you have data coming in to New Relic.